src/components/shared/subscribers/SubscribersModal.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+  Typography,
+  makeStyles,
+  Dialog,
+  DialogTitle,
+  DialogContent
+} from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  title: {
+    color: theme.palette.primary.dark
+  },
+  content: {
+    padding: 0,
+    maxHeight: '400px'
```

Comment:
what happens if the `content` rendered is more than `400px`, is scrolling enabled? I realise that `SubscribersList` is rendered here. I think you need to add an `overflow: auto` property to `content`, which wraps the `childComponent`